gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -198,6 +205,17 @@ private void handleAmbiguity(LatLng latLng) {
       }
     }
 
+    for (Map.Entry<MapFeature, List<LatLng>> json : polygons.entrySet()) {
```

Comment:
I was thrown off by the variable name `json` here, thought I hadn't had enough coffee yet :) Items in `entrySet` aren't json, they're map entries. Could you please rename to `entry` here? 

Also, to avoid confusion, please assign the key and value to local variables `mapFeature` and `vertices`.

Would you mind also updating the above code block in a separate commit as well?

Reply:
renamed.